libs/commons/ATD_string_wrap.ml

```diff
@@ -56,6 +56,19 @@ module Datetime = struct
     | Ok dt -> dt
     | Error s -> failwith (spf "wrong datetime format: %s" s)
 
+  let of_unix_int_time (timestamp : int64) (sign : [ `Plus | `Minus ])
```

Comment:
This is not the place for those functions. You can put this in a commons/Datetime_.ml
The goal of the module is to assist ATD when using the 'string wrap' annotations in our .atd files.